src/azul/terra.py

```diff
@@ -195,10 +197,27 @@ class TDRClient(SAMClient):
     A client for the Broad Institute's Terra Data Repository aka "Jade".
     """
 
+    def project_for_source(self, source: TDRSource) -> str:
+        """
+        Use the TDR API to find which Google Cloud project contains the source.
+        """
+        response = self._get_source_response(source)
+        assert response.status == 200
+        response = json.loads(response.data)
+        return response['dataProject']
+
     def check_api_access(self, source: TDRSource) -> None:
         """
         Verify that the client is authorized to read from the TDR service API.
         """
+        response = self._get_source_response(source)
+        assert response.status == 200
+        response = json.loads(response.data)
+        response.pop('tables', None)
+        log.info('TDR client is authorized for API access to %s: %r',
+                 source, response)
```

Comment:
Can we strip out even more information from the response? The only fields that seem important are the `dataProject` and `name`, but even those seem like they can be omitted. I certainly don't see the point in including the `source` or `relationship` fields. 

It might be worth removing the response entirely, if that is not an option then formatting the response so its more legible would be great.

Reply:
My interpretation is that none of the field values are actually used for anything, and that the point of logging the response JSON is to show/prove that the client is receiving a complete response. I wanted to err on the side of removing too little rather than too much but if you I are in agreement that this is still too much then I'll revert to an earlier and simpler approach that shows even less.